cmd/notification.go

```diff
@@ -559,11 +562,11 @@ func SaveListener(objAPI ObjectLayer, bucketName string, eventNames []event.Name
 		return err
 	}
 
-	return saveConfig(objAPI, configFile, data)
+	return saveConfig(ctx, objAPI, configFile, data)
 }
 
 // RemoveListener - removes HTTP client currently listening for events from listener.json.
-func RemoveListener(objAPI ObjectLayer, bucketName string, targetID event.TargetID, addr xnet.Host) error {
+func RemoveListener(ctx context.Context, objAPI ObjectLayer, bucketName string, targetID event.TargetID, addr xnet.Host) error {
```

Comment:
CRUD functions of notification and listener configuration files need to be private because object api has public methods.

Reply:
done.